README.md

```diff
@@ -14,3 +14,13 @@ Download the latest ChatBar.app file from the [releases page](https://github.com
 - Adding the chat icon to the control strip is handled by calls to a private, undocumented API(see [TouchBaer](https://github.com/a2/touch-baer)). 
 - The application simulates a keypress of the shift key regularly to disable the touchbar sleep function.
 - The connection is done uses websockets secured with TLS, provided by the [Starscream](https://github.com/daltoniam/Starscream) framework. 
+
+# Build
+```
+$ brew install carthage
+$ git clone https://github.com/vanHavel/ChatBar
+$ cd ChatBar
+$ carthage update --platform macOS
+$ open ChatBarProject/ChatBar.xcodeproj
```

Comment:
Add least for me, there is one additional step I need to run after carthage update: 
Copy the output from Carthage/Build/Mac/Starscream.framework to the project folder (ChatBar)

Reply:
Ohhh right, i forgot that. Was in a rush. Will fix that likely later today.

Reply:
updated the instructions and made them executable completely on the cli :)

Reply:
looks good!